lms/templates/admin/organization/show.html.jinja2

```diff
@@ -77,6 +80,18 @@
                         </div>
                     </div>
                 </li>
+                <li class="tab-panel" data-section-id="course">
```

Comment:
This is on organization and not applications instance.

That seem to contradict how things are organized on the DB but the DB schema is wrong. We should have only one grouping per org that gets linked to different applications instances that have launched it.

Now we get one course row per applications instance that has ever seen it which complicates things for querying /reporting.


If we also need to query them at the AI level we could use a couple of macros and have it in both places.